### DISCUSSION TITLE
[Question]: Access to cloudflare ip necessary?

### DISCUSSION THREAD

**Discussion by Schluggi** [danny-avila/LibreChat#1520] [Q&A]
### What is your question?

Hi,

I noticed the docker container wants to access a Cloudflare IP (104.16.28.34) on port 443 after startup. Is this necessary? And if yes, for what?

### More Details

\-

PS: The [Code of Conduct](https://github.com/danny-avila/LibreChat/blob/main/CODE_OF_CONDUCT.md) link at the issue template does work.

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by danny-avila** [comment]
That's not an IP familiar to me, what does your setup look like?

**Reply by danny-avila** [reply]
The only external network request off the top of my head that might be requested on startup is fetching the list of models from the AI provider. Seeing as the IP is located in San Francisco, this might be likely. There is no reference to port 443 anywhere in the app aside from nginx configuration
